### PR TITLE
added-animation-file

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+-+<!DOCTYPE html>
```
Comment (OSAMA478) [inline index.html]: i think this -+ shouldn't be there......

**Review by mmusk — APPROVED** [review]
LGTM

File: style.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    background: white;
+    box-shadow: 2px 2px 20px 6px rgba(173, 173, 173, 0.7) inset;
+}
+.container:hover .text:nth-child(4) span, .container:hover .text:nth-child(5) span{
+    color: #fff;
+    background: rgb(19, 136, 8);
+    box-shadow: 2px 2px 20px 6px rgba(15, 97, 8, 0.7) inset;
+}
+
+@media screen and (max-width:400px){
```
Comment (OSAMA478) [inline style.css]: if the max-width is 600px then it would be more visible to users with a screen width below 600px.

**Review by anask601 — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (murtazshaikh) [reply style.css]: okk I'll change it

File: index.html (hunk 1 above)
Comment (murtazshaikh) [reply index.html]: It came by default, well I have remove it.

**Review by syedabdullahali — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

File: style.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    font-size: 4em;
+    color: #000;
+
+    background: #585858;
+    box-shadow: 2px 2px 20px 6px rgba(48, 47, 47, 0.7) inset;
+    transition: 3.5s ease-in;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nehalskdev) [inline style.css:76]:
```suggestion
    display: grid;
    place-items: center;
```

**Pull request merged**